### ISSUE TITLE
使用3.7.5问题

### ISSUE THREAD

**Issue by FanBaobao132** [AriaLyy/Aria#513]
引用：annotationProcessor 'com.arialyy.aria:compiler:3.7.5'
    implementation 'com.arialyy.aria:ftpComponent:3.7.5' //# 如果需要使用ftp，请增加该组件
    api('com.arialyy.aria:aria-core:3.7.5'){
        exclude group: 'androidx.appcompat.app'
    }
报错：
ERROR: Failed to resolve: com.arialyy.aria:aria-core:3.7.5

**Comment by FanBaobao132** [comment]
名字写错了e
